README.md

```diff
@@ -35,6 +35,23 @@ To release a new version with your changes, run `yarn changeset add` and follow
 
 When your PR is merged, changeset will analyse the changes and create a PR to release the new version.
 
+For an efficient and secure process, a script is available to update the version of commercial lib in Frontend to your specified version. You can find the script [here](https://gist.github.com/chrislomaxjones/2663b9bcafd0ad61857b9cd6191c1a78).
```

Comment:
I would recommend adding this script to the repo and linking to it here, rather than linking to Chris' gist.